sdk/communication/communication-chat/swagger/README.md

```diff
@@ -67,3 +67,13 @@ directive:
   transform: >
     $["x-ms-enum"].modelAsString = false;
 ```
+
+### Set PolicyViolationMessageState Model as string false
```

Comment:
Why when we do the renaming, the rename is not applied to Chat gateway? Chat Gateway keep using `state` but only the client side using `result`? Is this what we want?